src/main/java/org/gridsuite/casevalidation/server/LoadFlowCaseValidationService.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.casevalidation.server;
+
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@Service
+class LoadFlowCaseValidationService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoadFlowCaseValidationService.class);
+
+    @Autowired
+    private LoadFlowService loadFlowService;
+
+    LoadFlowCaseValidationReport validate(UUID networkUuid) {
+        //Validation with default loadflow parameters
+        LoadFlowParameters params = new LoadFlowParameters()
+                .setTransformerVoltageControlOn(true)
+                .setSimulShunt(true)
+                .setDistributedSlack(true)
+                .setBalanceType(LoadFlowParameters.BalanceType.PROPORTIONAL_TO_GENERATION_P_MAX)
+                .setReadSlackBus(true)
+                .setVoltageInitMode(LoadFlowParameters.VoltageInitMode.DC_VALUES);
+        if (validate(networkUuid, params)) {
+            return new LoadFlowCaseValidationReport(true, LoadFlowCaseValidationReport.Status.CONVERGED_ON_1ST_LF);
+        }
+
+        //Validation with relaxed loadflow parameters
+        params.setTransformerVoltageControlOn(false);
+        params.setSimulShunt(false);
+
+        boolean isLoadFlowOk = validate(networkUuid, params);
+        return new LoadFlowCaseValidationReport(isLoadFlowOk, isLoadFlowOk ? LoadFlowCaseValidationReport.Status.CONVERGED_ON_2D_LF : LoadFlowCaseValidationReport.Status.FAILED);
+    }
+
+    boolean validate(UUID networkUuid, LoadFlowParameters params) {
+        LoadFlowResult result = loadFlowService.run(networkUuid, params);
+        LOGGER.info("Loadflow validation for case {} with loadflow parameters : {}", networkUuid, params);
+        boolean isLoadFlowOk = isMainComponentConverging(result);
+        LOGGER.info("Loadflow status: {}", isLoadFlowOk ? "OK" : "KO");
+        LOGGER.info("Loadflow metrics: {}", result.getMetrics());
```

Comment:
Don't write metrics this is a useless and impl specific infos

Reply:
done